test/__mocks__/react-i18next.ts

```diff
@@ -0,0 +1,12 @@
+const reacti18next: any = jest.createMockFromModule('react-i18next');
+
+reacti18next.useTranslation = () => ({
+    t: (str: string) => str,
+    i18n: {
+      changeLanguage: () => new Promise(() => {}),
+    },
+  });
+
+module.exports = reacti18next;
+
+export default {};
```

Comment:
[curious] was the `export default {}` actually required?

Reply:
Without it, I got this error:

> 'react-i18next.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.ts(1208)